src/h5web/toolbar/controls/ToggleBtn.tsx

```diff
@@ -11,22 +11,32 @@ interface Props {
   value: boolean;
   onChange: () => void;
   disabled?: boolean;
-}
+};
 
 function ToggleBtn(props: Props): ReactElement {
-  const { label, small, raised, value, onChange, disabled } = props;
-  const { icon: Icon, iconOnly } = props;
+  const {
+    label,
+    icon: Icon,
+    iconOnly,
+    small,
+    raised,
+    value,
+    onChange,
+    disabled,
+    ...ariaAttrs
+  } = props;
 
   return (
     <button
       className={styles.btn}
       type="button"
-      aria-label={iconOnly ? label : undefined}
-      aria-pressed={value}
       onClick={onChange}
       disabled={disabled}
       data-small={small || undefined}
       data-raised={raised || undefined}
+      aria-label={iconOnly ? label : undefined}
+      aria-pressed={value}
+      {...ariaAttrs}
```

Comment:
This is so I can pass `aria-expanded` and `aria-controls` to the "Edit domain" toggle button.